sdk/tables/data-tables/test/public/tableclient.spec.ts

```diff
@@ -619,4 +619,35 @@ describe(`TableClient`, () => {
       });
     });
   });
+
+  describe("tracing", () => {
```

Comment:
Originally I had one test per client method, which may be better / clearer. Alternatively, this is kinda neat in that you're just telling a story of a client invoking various operations and then ensuring tracing spans tell the same story. Not sure which is nicer - feel free to sound off in the comments (and don't forget to like, comment, and subscribe 😆 )

Reply:
I liked, commented and subscribed to the idea :D